eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Utxos.hs

```diff
@@ -232,12 +265,35 @@ scriptsNotValidateTransition = do
   let txb = body tx
   sysSt <- liftSTS $ asks systemStart
   ei <- liftSTS $ asks epochInfo
+  let ~traceIdentifier =
+        (castHash $ hashWithSerialiser @ShortHash toCBOR txb)
+          `xor` (castHash $ hashWithSerialiser @ShortHash toCBOR utxo)
```

Comment:
Erm.... this is hashing the _entire_ UTxO set. Please do not do this!

Reply:
right, i guess the txbody should be good enough to uniquely identify each call? i wasn't really concerned with performance since this should only get evaluated with `-eventlog` is passed and i was not aware of how insane this was lol